src/DOMUpdates.js

```diff
@@ -1,13 +1,18 @@
 import loginHandler from './loginHandler.js';
 import moment from 'moment';
 
-const domUpdates = {
+class DOMUpdates {
+  constructor() {
+    this.currentUser = null;
+    this.date = null;
```

Comment:
I notice the change to an object here - I'll look but if I forget to follow up - does DOMUpdates get instantiated more than once? If not it can remain an object and still have currentUser and date.

Reply:
I found it in the login-handler.
I don't think storing currentUser as "customer20" is a bad thing at all. Your high degree of separation between data and functionality is admirable. Looking into this was also an interesting exploration of how differently coders can think.